pkg/kubectl/cmd/label.go

```diff
@@ -25,12 +25,14 @@ import (
 	cmdutil "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/resource"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
 	"github.com/spf13/cobra"
 )
 
 const (
 	label_long = `Update the labels on a resource.
 
+A valid label value is consisted of letters and/or numbers with a max length of %[1]d characters.
```

Comment:
Labels can also contain `-`, `.`, and`_`, which is why I was suggesting putting the regex into the help string. Rule is [here](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/util/validation.go#L27).